src/Type/ObjectType/EnqueuedScript.php

```diff
@@ -21,19 +21,26 @@ public static function register_type() {
 			'description' => __( 'Script enqueued by the CMS', 'wp-graphql' ),
 			'interfaces'  => [ 'Node', 'EnqueuedAsset' ],
 			'fields'      => [
-				'id'  => [
+				'id'      => [
 					'type'    => [
 						'non_null' => 'ID',
 					],
 					'resolve' => function ( $asset ) {
 						return isset( $asset->handle ) ? Relay::toGlobalId( 'enqueued_script', $asset->handle ) : null;
 					},
 				],
-				'src' => [
+				'src'     => [
 					'resolve' => function ( \_WP_Dependency $script ) {
 						return isset( $script->src ) && is_string( $script->src ) ? $script->src : null;
 					},
 				],
+				'version' => [
+					'resolve' => function ( \_WP_Dependency $script ) {
+						global $wp_scripts;
+
+						return isset( $script->ver ) && is_string( $script->ver ) ? (string) $script->ver : $wp_scripts->default_version;
+					},
```

Comment:
along the lines of my other comment, when we default to the current WordPress version, perhaps we should hash it to make it an arbitrary value? maybe something like:

```php
$version = \WPGraphQL\Router::$route . $wp_scripts->default_version;
$hashed_version = md5( $version );
```